src/libraries/System.Reflection.Metadata/src/System.Reflection.Metadata.csproj

```diff
@@ -6,6 +6,7 @@
     <CLSCompliant>false</CLSCompliant>
     <PackageTargetFramework Condition="'$(TargetFramework)' == 'netstandard1.1'">netstandard1.1;portable-net45+win8</PackageTargetFramework>
     <Configurations>$(NetCoreAppCurrent)-Debug;$(NetCoreAppCurrent)-Release;netstandard2.0-Debug;netstandard2.0-Release;netstandard1.1-Debug;netstandard1.1-Release</Configurations>
+    <_ExcludeAPIList>$([MSBuild]::NormalizePath('$(MSBuildProjectDirectory)', '..', 'ref', 'ReferenceSourceExcludeApi.txt'))</_ExcludeAPIList>
```

Comment:
I would add this in Directory.Build.targets and condition setting the property if the file exists. That way it will be a light-up feature, that if you have that file under the ref/ directory, it will automatically pass it down to `GenAPI`.